pom.xml

```diff
@@ -193,13 +213,82 @@
                     <target>1.8</target>
                 </configuration>
             </plugin>
+            <!-- Adds source and resource directories to our build -->
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>1.9.1</version>
+                <executions>
+                    <!-- Add a new source directory to our build -->
+                    <execution>
+                        <id>add-integration-test-sources</id>
+                        <phase>generate-test-sources</phase>
+                        <goals>
+                            <goal>add-test-source</goal>
+                        </goals>
+                        <configuration>
+                            <!-- Configures the source directory of our smoke tests -->
+                            <sources>
+                                <source>src/smoke-test/java</source>
+                            </sources>
+                        </configuration>
+                    </execution>
+                    <!-- Add a new resource directory to our build -->
+                    <execution>
+                        <id>add-integration-test-resources</id>
+                        <phase>generate-test-resources</phase>
+                        <goals>
+                            <goal>add-test-resource</goal>
+                        </goals>
+                        <configuration>
+                            <!-- Configures the resource directory of our smoke tests -->
+                            <resources>
+                                <resource>
+                                    <directory>src/smoke-test/resources</directory>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
             <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.18.1</version>
+                <configuration>
+                    <!-- Skips unit tests if the value of skip.unit.tests property is true -->
+                    <skipTests>${skip.unit.tests}</skipTests>
+                    <excludes>
+                        <exclude>**/*ST*.java</exclude> <!-- Exludes smoke tests when unit tests are run -->
+                        <exclude>**/*IT*.java</exclude> <!-- Exludes integration tests when unit tests are run -->
```

Comment:
Nå finnes det ingen *IT tester i prosjektet så man kan i grunn fjerne den ene excluden. (Eller så kan den være der slik at den uansett ignorerer kommende IT-tester) 🎱